voting_schemes/electionguard/python-wrapper/src/bulletin_board/electionguard/bulletin_board.py

```diff
@@ -14,7 +14,7 @@
 from electionguard.type import GUARDIAN_ID
 from electionguard.utils import get_optional
 
-from .common import Content, Context, ElectionStep, Wrapper
+from .common import Content, Context, ElectionStep, Wrapper, async_wrap
```

Comment:
Is `async_wrap` being used in this file?

Reply:
looks like not. This also comes from the original PR. Do yo want me to remove it?

Reply:
Yes, please. As far as I see this is imported but not used, so it seems like its not doing anything. If that's not the case and there's some failing specs after this change, we can reintroduce it.